be/src/column/const_column.cpp

```diff
@@ -56,8 +56,8 @@ void ConstColumn::fill_default(const Filter& filter) {
     CHECK(false) << "ConstColumn does not support update";
 }
 
-Status ConstColumn::update_rows(const Column& src, const uint32_t* indexes) {
-    return Status::NotSupported("ConstColumn does not support update");
+void ConstColumn::update_rows(const Column& src, const uint32_t* indexes) {
+    CHECK(false) << "ConstColumn does not support update_rows";
```

Comment:
If some query has a bad plan, this code is executed, which will cause the process to crash.
But we don't plan to crash the whole process because of one bad case.

Reply:
> If some query has a bad plan, this code is executed, which will cause the process to crash. But we don't plan to crash the whole process because of one bad case.

fixed by throwing an exception here and catching it by caller.